Neutaro/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "Neutaro",
+  "assets": [
+    {
+      "description": "The native token of Neutaro",
+      "denom_units": [
+        {
+          "denom": "uneutaro",
+          "exponent": 0
+        },
+        {
+          "denom": "neutaro",
+          "exponent": 6
+        }
+      ],
+      "base": "uneutaro",
+      "name": "neutaro",
+      "display": "neutaro",
+      "symbol": "$ntmpi",

```

Comment:
Please use all caps for ticker symbols and exclude the `$`

Reply:
Done